modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -86,7 +86,7 @@ spec:
      - secureforward-offcluster
 ----
 <1> The name of the log forwarding CR must be `instance`.
-<2> Parameter to enable log forwarding. Set to `true` to enable log forwarding.
+<2> Parameter to enable log forwarding. Set to `true` to enable log forwarding. Do not set to `false`. Setting this parameter to `false` prevents cluster logging from sending logs to the specified endpoints *and* to default internal {product-title} Elasticsearch instance.
```

Comment:
This is incorrect.  It should be something like:

```
<2> Parameter to disable the operator from creating log forwarding config to the default cluster logging managed store. Set to `true` to disable generation of default store logforwarding config. Note: this change assumes administrators will define logforwarding for default storage.
```

Reply:
@jcantrill Thanks for the clarification. I meant here to use `true` to enable the Log Forwarding feature, to keep the wording simple. I can see where that would be confusing as set the parameter to enable forwarding is wrong, as log forwarding is enabled by default. 
I put wording similar to your further up in the file. Again to keep it simple here is the call-out. 
`<2> Parameter to disable the default log forwarding behavior. Set to `true` to enable log forwarding to the specified outputs.`
The default behavior is explained above.